src/features/deposit/components/DepositWidget.tsx

```diff
@@ -17,23 +18,22 @@ import {
   type DepositFormNearValues,
 } from "./Form/DepositFormNear"
 
-type DepositWidgetProps = {
-  accountId: string
-  signAndSendTransactionsNear: (transactions: Transaction[]) => void
-}
-
 export const DepositWidget = ({
+  tokenList,
   accountId,
-  signAndSendTransactionsNear,
+  sendTransaction,
 }: DepositWidgetProps) => {
   const [formType, setFormType] = useState<DepositFormType | null>(null)
   const [blockchain, setBlockchain] = useState<BlockchainEnum | null>(null)
   const [asset, setAsset] = useState<BaseAssetInfo | null>(null)
   const depositFormNearMethods = useForm<DepositFormNearValues>()
 
+  assert(tokenList.length > 0, "Token list must have at least 1 token")
```

Comment:
What is the purpose of this check?

Reply:
To deposit we need at least one token in the list, otherwise it won't be able to do anything.

Reply:
Okay, such check assumes you need it to verify assumption about data, but in your case the list will be just empty. And it won't cause any issues.